### ISSUE TITLE
"Better" matching communitheme colors?

### ISSUE THREAD

**Issue by wandrewkeech** [snwh/suru-icon-theme#31]
Not a bug, more of a question about design sensibility. I have made a couple branches in a fork for matching the elements of Tweaks and Software Store to the Suru colors as they seem to be stabilizing in the community gtk theme. Is your feeling that the icons are better with Ubuntu Orange to have a more cohesive "system" look, distinct from the design elements which are communicating system state, or is it better to have more observably memetic design for the app icons?

**Comment by snwh** [comment]
Is the GTK theme not using the original Suru colour palette? I have a copy in this project: https://github.com/snwh/suru-icon-theme/blob/master/src/Suru.gpl

**Comment by snwh** [comment]
Also, I have *no idea* what you're asking me here, please clarify (in simpler terms).

**Comment by wandrewkeech** [comment]
It's a question about, first, whether this looks good, and second, whether it's correct to do before sending you a PR (correctness in the sense of more orange = more systemish, more cohesive):

![software-store](https://user-images.githubusercontent.com/7744003/36929967-4b83d15a-1e4d-11e8-9c22-0f3d91896a3c.png)
![tweaks-app](https://user-images.githubusercontent.com/7744003/36929969-4d8a1afe-1e4d-11e8-9d5b-47ab35245c93.png)

I also want to submit more (new) icons, but don't want to bombard you with trash you won't like. Right now I'm thinking deja-dup, update-manager, and software-properties-gtk (as standard Ubuntu desktop fare)

**Comment by snwh** [comment]
You are free to submit pull requests for review at any time. :) 

The blue you've added to the Software icon takes away from the cohesiveness of that icon's design and uncessarily draws the eye.

I'd be happy to change the switch color in the tweaks app icon, if the green is what is being used in the GTK theme.

~Deja-dup will fall under third-party applications~ Deja dup is a GNOME app, it could probably do with an icon, and if 18.04 is going with GNOME Software, I doubt update-manager and software-properties will be shipped.

**Comment by ghost** [comment]
> The blue you've added to the Software icon takes away from the cohesiveness of that icon's design and uncessarily draws the eye.

The progressbar in communitheme is blue. Thats why he used blue. It fits more to the gtk-theme.

EDIT: When the switch is activated the knob is on the right side. The Icon shows it on the left side when activated. Maybe this can be solved by mirroring the icon.
![bildschirmfoto von 2018-03-08 12-02-33](https://user-images.githubusercontent.com/36449926/37147996-21f7eca6-22c9-11e8-8e7d-01a1bb03de6d.png)
